core/templates/services/local-storage.service.ts

```diff
@@ -82,7 +82,7 @@ export class LocalStorageService {
     if (this.isStorageAvailable()) {
       let draftDict = ExplorationDraft.toLocalStorageDict(
         changeList, draftChangeListId);
-      this.storage.setItem(localSaveKey, JSON.stringify(draftDict));
+      (<Storage> this.storage).setItem(localSaveKey, JSON.stringify(draftDict));
```

Comment:
Why do we need the casting?

Reply:
ah, okay, can we add a comment?

Reply:
added comments but I feel like they may be redundant.